src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -403,19 +487,43 @@ def run_acr_build(self, dockerfile, source, quiet=False, build_from_source=False
         image_name = self.image if self.image is not None else self.name
         from datetime import datetime
 
-        now = datetime.now()
-        # Add version tag for acr image
-        image_name += ":{}".format(
-            str(now).replace(" ", "").replace("-", "").replace(".", "").replace(":", "")
-        )
-
         self.image = self.registry_server + "/" + image_name
 
+        now = datetime.now()
+        tag_now_suffix = str(now).replace(" ", "").replace("-", "").replace(".", "").replace(":", "")
```

Comment:
It is suggested to add a comment for adding an example or explaining this processing.

Reply:
Updated: added comment for better explanation